PluginGenerator/Resources/Rules/Rules.pom

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <dependencies>
+
+    <!-- Additional dependencies required by plugins that use rules -->
+
+    <dependency>
+      <groupId>org.codehaus.sonar.sslr-squid-bridge</groupId>
+      <artifactId>sslr-squid-bridge</artifactId>
+      <version>2.6</version>
+    </dependency>
+    
+  </dependencies>
+
+</project>
```

Comment:
We wouldn't have completely separate pom files if we were using Maven directly, However, we aren't: we're just using the pom as a convenient format to specify the dependencies.